tests/utils/checks.ts

```diff
@@ -15,6 +15,7 @@ export function checkIfIsIncremented(initialIndex: number, finalIndex: number) {
   console.log('INITIAL:', initial);
   console.log('FINAL:', final);
   const isIncremented = final === initial + 1;
+  saved.length = 0;
```

Comment:
This resets the saved variable. Before the state inside saved would be carried through onto each test rendering it incorrect. 

i.e. 
before: [123, 124, 2, 3]

where 123, 124 is the quickstart test and 2, 3 is the intro to sway test. Since the function always looks at the first two in the array, tests will always run true if they run after quickstart. 

after: [2, 3]

Reply:
Can you change this to `saved = [];` and use `let` to declare the save array instead?